scm-info/com/atlassian/event/_artifact/atlassian-event/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://code.engineering.redhat.com/gerrit/atlassian/atlassian-event.git"
```

Comment:
We need a private (internal) repo for these ones, we should not be referencing internal repositories in a public repo.